app.js

```diff
@@ -10,27 +10,31 @@ app.get('/', function(_req, res) {
   res.send({ message: '👋 Hi, this is the impersonation-service 🕵' });
 });
 
-app.get('/impersonations/current', async function(req, res) {
+app.get('/impersonations/current', async function (req, res, next) {
   const muSessionId = req.get('mu-session-id');
+  if (!muSessionId)
+    return next({ message: 'Session header is missing' });
```

Comment:
I copied this from the [acmidm-login service](https://github.com/lblod/acmidm-login-service/blob/266314133ea9761a35e471706e49c3dfef254181/app.js#L152-L153) but I'm not sure it is needed.

I ran into this when sending requests without a mu-session-id, but I'm not sure if that happens in real apps. If we do want it, we probably want it in all the endpoints?

Reply:
The `acmidm-login-service` seems to handle it on each endpoint. It's a good point to have a request handler, but this service doesn't have much, so I think we can do it manually on each endpoint in this case.

Reply:
I've reverted the change for now since it's not strictly needed, and was unrelated to what I actually wanted to change. I we want these checks we can add them in a separate PR.